src/components/Model/model.utils.ts

```diff
@@ -39,6 +39,7 @@ const modelQueryParamSchema = z
       (val) => (Array.isArray(val) ? val : [val]),
       z.array(z.enum(constants.baseModels))
     ),
+    clubId: z.number().optional(),
```

Comment:
Since this is coming from the query string, we should probably coerce the type to avoid parsing errors
```suggestion
    clubId: z.coerce.number().optional(),
```